src/components/pages/datasets/pep/PepList.jsx

```diff
@@ -69,10 +69,10 @@ const PepList = () => {
     },
     {
       header: t('lastPlaceOfWork'),
-      prop: 'last_employer',
+      prop: i18n.language === 'uk' ? 'last_employer' : 'last_employer_en',
       width: '20%',
       filter: {
-        name: 'last_employer',
+        name: i18n.language === 'uk' ? 'last_employer' : 'last_employer_en',
```

Comment:
we dont have filter for `last_employer_en`

Reply:
filter added in 3v-workspace/Data_converter#880

Reply:
filter disappears when locale is en